pkg/apis/flowcontrol/v1beta3/defaults.go

```diff
@@ -51,8 +68,13 @@ func SetDefaults_ExemptPriorityLevelConfiguration(eplc *v1beta3.ExemptPriorityLe
 	}
 }
 
-func SetDefaults_LimitedPriorityLevelConfiguration(lplc *v1beta3.LimitedPriorityLevelConfiguration) {
```

Comment:
keep this function defined, just move the NominalConcurrencyShares bit to SetDefaults_PriorityLevelConfiguration